std/typecons.d

```diff
@@ -3059,25 +3061,31 @@ struct Nullable(T)
         _isNull = false;
         return this;
     }
+    /// ditto
+    Nullable opAssign()(typeof(null) value)
+    {
+        if(value !is null)
+            assert(false);
+        nullify();
+        return this;
+    }
 
     /**
      * If this `Nullable` wraps a type that already has a null value
      * (such as a pointer), then assigning the null value to this
-     * `Nullable` is no different than assigning any other value of
-     * type `T`, and the resulting code will look very strange. It
-     * is strongly recommended that this be avoided by instead using
-     * the version of `Nullable` that takes an additional `nullValue`
-     * template argument.
+     * `Nullable` is no different than assigning $(REF nullify) ing it.
```

Comment:
This wording seems pretty awkward. Did you mean to remove the word "assigning" as well?

Reply:
Yea, I seem to have forgotten to remove "assigning". Thank you! :)